appliance/roles/common/vars/main.yml

```diff
@@ -30,18 +30,18 @@ common_packages:
   - lvm2
   - gettext
 
-# Usage: pre_requisite.yml
+# Usage: tasks/pre_requisite.yml
 internet_delay: 0
 internet_timeout: 1
 hostname: github.com
 port_no: 22
 os_name: CentOS
-os_version: '8'
+os_version: '8' 
```

Comment:
@j0hnL @sujit-jadhav since we will be using CentOS7 for the first version, should we roll this `os_version` back?

Reply:
@j0hnL @lwilson , This code is written in Sprint 3, so it is specific to CentOS 8. In Sprint 4 we will be making changes for CentOS 7, we will modify code again.

Reply:
Sounds good @sujit-jadhav. I've approved this PR.